kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/update-domain.sh

```diff
@@ -145,6 +145,10 @@ function createDomainConfigmap {
   if [ -d "${scriptDir}/common" ]; then
     cp ${scriptDir}/common/* ${externalFilesTmpDir}/
   fi
+  if [ -d "${scriptDir}/../../common" ]; then
```

Comment:
Can this script be used for wlst ? If not, do we give proper error message ?

Reply:
The copyright information must be changed to 
 ... Copyright (c) 2018, 2021, Oracle and/or its affiliates.

Reply:
The description section on this script and the usage info is exactly matches with create-domain.sh.  It should be treated as different purpose

$ create-domain.sh -h  
operator(pani-1)>sh create-domain.sh -h
usage: create-domain.sh -o dir -i file [-e] [-v] [-h]
  -i Parameter inputs file, must be specified.
  -o Output directory for the generated yaml files, must be specified.
  -m model file name, optional.
  -e Also create the resources in the generated yaml files, optional.
  -v Validate the existence of persistentVolumeClaim, optional.
  -h Help
$ update-domain.sh -h
usage: ./update-domain.sh -o dir -i file [-e] [-v] [-h]
  -i Parameter inputs file, must be specified.
  -o Output directory for the generated yaml files, must be specified.
  -m model file name, optional.
  -e Also create the resources in the generated yaml files, optional.
  -v Validate the existence of persistentVolumeClaim, optional.
  -h Help

Reply:
Not sure what you are saying here. the options are the same for both create and update scripts.

Reply:
But the purpose are different,  Are those alternative to each other.

Reply:
I fixed the update domain script to run even if the domain was created by wlst